backbone/src/main/java/org/researchstack/backbone/ui/task/TaskActivity.kt

```diff
@@ -48,6 +48,7 @@ open class TaskActivity : AppCompatActivity(), PermissionMediator {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        window.setFlags(WindowManager.LayoutParams.FLAG_SECURE, WindowManager.LayoutParams.FLAG_SECURE)
```

Comment:
You are missing the IF condition here. Check how this is applied in other activities.

Reply:
it should be something like: 

`if (BuildConfig.USE_SECURE_FLAG) {
            window.setFlags(
                WindowManager.LayoutParams.FLAG_SECURE,
                WindowManager.LayoutParams.FLAG_SECURE
            )
        }`

Reply:
oh I thought this should be applied to all flavors, ok I'll edit that now